### ISSUE TITLE
Links not showing preview images

### ISSUE THREAD

**Issue by fernweh01** [gnotaras/wordpress-add-meta-tags#40]
Prior to using the Add-Meta-Tags plugin, whenever I would share links to my content on Google+ and Tumblr the links would automatically include my post's featured image. Now since using the plugin there's no longer a preview image accompanying my links. Am I missing something? Is there a setting I should have ticked?

**Comment by gnotaras** [comment]
Hi and thanks for your question.

As long as a meta tag, which is supported by the social network, contains the link to the featured image, which in turn is in compliance with the image specs of each social network, the image should be displayed. In this case. these two social networks support Opengraph meta tags (Google also supports Schema.org). So, please make sure you enable those metadata generators in the plugin settings in the WordPress admin interface (Settings -> Metadata).

As a general note, the responsibility of Add-Meta-Tags ends with the generation of standards compliant metadata on your web site. How this metadata is used by the social networks and other services is beyond the plugin's control.

So, in order to be able to better understand the problem, I would appreciate some more feedback about the shared image (especially dimensions) and about the generated metadata on your end. Also, please make sure there are no htaccess rules that prevent image hot linking. Moreover, it has been reported in the past (about LinkedIn) that security plugins blocked that social network's bot, so the metadata was not read.

In case of G+ and tumblr, all that is required is an `og:image` meta tag which points to the featured image. As long as you have set a featured image and as long as the Opengraph generator has been enabled in the plugin settings, sharing on those networks should work as expected.

Please feel free to post more feedback or questions.

Kind Regards,
George

**Comment by fernweh01** [comment]
Hi George,

Thanks so much for your helpful reply. I just enabled the Opengraph and Schema.org settings and my images are showing up perfectly now! The world of metadata and meta tags is completely new to me, so I appreciate you taking the time to write such an informative response. So far I love the plugin. It's been so much easier to use than the current SEO plugins available. Even though I've only been using it for a few days, I've already noticed a great improvement in how my site and posts appear in search engine results thanks to the ability to customize the meta titles and descriptions. I still have a lot to learn about metadata, but thanks to your plugin and advice I'm now off to a good start!
